.github/workflows/danger.yml

```diff
@@ -2,16 +2,7 @@ name: 'Danger JS'
 on:
   pull_request:
     types:
-      [
-        assigned,
-        edited,
-        labeled,
-        opened,
-        reopened,
-        synchronize,
-        unassigned,
-        unlabeled,
-      ]
```

Comment:
@rociobaena is there a reason we removed the `edited` event type from this list? That will make Danger not run again if we edit the comment in our PR. This means that if a PR fails because it is missing a comment about dependencies, editing the PR comment to fix it will not trigger Danger to run again. I don't think we want that.